Patch to remove "duplicate" bits of logs from opkg output, which 
massively simplifies do_rootfs logs. The reason is we get unflushed 
data passed to the children and duplicated.

RP - 26/1/10

Index: trunk/libbb/gz_open.c
===================================================================
--- trunk.orig/libbb/gz_open.c	2010-01-26 23:12:10.000000000 +0000
+++ trunk/libbb/gz_open.c	2010-01-26 23:12:17.000000000 +0000
@@ -38,6 +38,12 @@
 		perror_msg("pipe");
 		return(NULL);
 	}
+
+    /* If we don't flush, we end up with two copies of anything pending, 
+       one from the parent, one from the child */
+    fflush(stdout);
+    fflush(stderr);
+
 	if ((*pid = fork()) == -1) {
 		perror_msg("fork");
 		return(NULL);
